src/openexr.imageio/exrinput.cpp

```diff
@@ -113,9 +113,31 @@ static std::map<std::string, std::string> exr_tag_to_oiio_std {
     { "pixelAspectRatio", "PixelAspectRatio" },
     { "xDensity", "XResolution" },
     { "expTime", "ExposureTime" },
+    { "aperture", "FNumber" },
+    { "isoSpeed", "Iso", },
+    { "focus", "Focus", },
+    { "cameraCCTSetting", "WhiteBalanceTemperature", },
+    { "cameraTintSetting", "WhiteBalanceTint", },
+    { "nominalFocalLength", "FocalLengthNominal", },
+    { "pinholeFocalLength", "FocalLengthPinhole", },
+    { "effectiveFocalLength", "FocalLengthEffective", },
+    { "entrancePupilOffset", "EntrancePupil", },
+    { "sensorPhotositePitch", "PhotositeSensor", },
+    { "cameraMake", "Make", },
+    { "cameraModel", "Model", },
+    { "cameraSerialNumber", "SerialNumber", },
+    { "cameraFirmwareVersion", "FirmwareVersion", },
+    { "cameraUuid", "Uuid", },
+    { "cameraLabel", "Label", },
+    { "lensMake", "LensMake", },
+    { "lensModel", "LensModel", },
+    { "lensSerialNumber", "LensSerialNumber", },
+    { "lensFirmwareVersion", "LensFirmwareVersion", },
+    { "longitude", "Longitude", },
+    { "latitude", "Latitude", },
+    { "altitude", "Altitude", },
```

Comment:
What exactly are you trying to do here? What is the reason for making a new standard OIIO name "LensModel" but then saving it in EXR files as "lensModel"? 

If you want to propose adding a new "standard OIIO metadata" that corresponds to the OpenEXR one, why not just use the same name as OpenEXR and not have to do any name translation at all?

I'll say it again:

> This table is for translating OpenEXR standard metadata names to and from OTHER NAMES that we chose for OIIO to use across all formats, and which therefore we cannot easily change.

For any NEW names that we wish OIIO to use across all formats, we can pick any name we want, so why wouldn't we pick the name OpenEXR is already using?

Reply:
done.